### ISSUE TITLE
Str representation of mpq is different from Fractions

### ISSUE THREAD

**Issue by marcogario** [aleaxit/gmpy#113]
Hi, with the following example you can see a discrepancy between the behavior of ```str``` of mpq and Fractions in python 2.7 (I tried on 2.7.9). The difference is the trailing L at the end of the numerator:

```
import gmpy2
a = gmpy2.mpq(2**200, 7)
print(str(a))
# '1606938044258990275541962092341162602522202993782792835301376L/7'
from fractions import Fraction
b = Fraction(2**200, 7)
print(str(b))
# '1606938044258990275541962092341162602522202993782792835301376/7'
```

This seems to be controlled by an explicit condition in the code testing for PY2 (see [1]). I think that this has to do with the difference between ```repr``` and ```str``` in py2, as discussed in [2].

[1] https://github.com/aleaxit/gmpy/blob/8a5de1d3c32ba9f7e5e7c873d5ff44fda46ba561/src/gmpy2_convert_gmp.c#L962

[2] http://stackoverflow.com/questions/24059842/l-suffix-in-long-integer-in-python-3-x

**Comment by casevh** [comment]
A fix was pushed to the the gmpy2-2.0.9 branch.

Thanks again for all the testing.

**Comment by agroce** [comment]
Sure -- still running, but nothing showing up today except instances of https://bugs.python.org/issue27870
